server/src/main/java/org/opensearch/index/compositeindex/datacube/startree/utils/SequentialDocValuesIterator.java

```diff
@@ -36,7 +36,7 @@ public class SequentialDocValuesIterator {
     /**
      * The id of the latest document.
      */
-    private int docId;
+    private int docId = -1;
```

Comment:
Few of the getDocValue/setDocValue methods don't seem to be used. Can we remove them? Also, if we don't need any additional state (i.e. docValue, docId), lets get rid of it.